frontend/src/pages/EditRolesPage/components/buttons/EditRolesButton.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { Profiler } from 'react';
```

Comment:
why do we need a Profiler import here?

Reply:
This is unused and I plan on removing this upon cleaning up my changes